lib/cisco_node_utils/cmd_ref/vlan.yaml

```diff
@@ -44,6 +44,44 @@ name:
   set_context: ["vlan %d"]
   set_value: "%s name %s"
 
+private_vlan_association:
+  _exclude:
```

Comment:
You should be able to use the more concise format her as well: `_exclude: [N8k]`